db/migrate/20230620113412_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :bio
+      t.integer :posts_counter
+
+      t.timestamps
```

Comment:
- [ ] Based on the project ERD diagram, the column photo should be included in the User model. Kindly modify this user model using another migration file to add a column named `photo`.